trade-finance-manager-ui/server/constants/entra-id-user.ts

```diff
@@ -0,0 +1,23 @@
+export const ALL_ENTRA_ID_USER_GROUPS = [
+  'AZURE_SSO_GROUP_UNDERWRITING_SUPPORT',
+  'AZURE_SSO_GROUP_UNDERWRITER_MANAGERS',
+  'AZURE_SSO_GROUP_UNDERWRITERS',
+  'AZURE_SSO_GROUP_RISK_MANAGERS',
+  'AZURE_SSO_GROUP_BUSINESS_SUPPORT',
+  'AZURE_SSO_GROUP_PIM',
+  'AZURE_SSO_GROUP_PDC_READ',
+  'AZURE_SSO_GROUP_PDC_RECONCILE',
+] as const;
+
```

Comment:
Should this file maybe be called `entra-id-user-groups`?

Reply:
renamed